instructor_notes/openscpca/04_code-review-walkthrough.md

```diff
@@ -0,0 +1,36 @@
+
+# Instructor Notes: Code Review Walkthrough
+
+
+## Learning goals
+
+- Get familiar with a typical code review process in the OpenScPCA project
+- Understand the components of a Pull Request (PR) in the OpenScPCA project
+- See what a typical review process might look like
+
+## Activity/Demo
+
+1. Start with the showing the following open issue: https://github.com/AlexsLemonade/OpenScPCA-analysis/issues/374
+    - Note the link back to the discussion topic. (Do we want to show the form for submitting a new issue and/or discussion topic here?)
+    - Note links to other issues (we probably want to do this from a browser _without_ ZenHub)
+    - Note assignees as a thing?
+
+2. Show the Pull Request template:
+    - Open a new tab to *your* fork of the OpenScPCA-analysis repository
+    - Have a branch with some changes ready to go, and navigate to that branch
+    - Start to create a PR from that branch back to origin/main
+    - Show the pull request template: note that this one is _not_ a form, but a Markdown file that you will need to edit
```

Comment:
Might point out the checkboxes at the bottom, and that those are easiest to check off once the PR is filed.